pkg/api/k8s_versions.go

```diff
@@ -583,7 +583,7 @@ func getK8sVersionComponents(version, kubernetesImageBaseType string, overrides
 			common.CloudControllerManagerComponentName:        azureCloudControllerManagerImageReference,
 			common.CloudNodeManagerAddonName:                  azureCloudNodeManagerImageReference,
 			common.WindowsArtifactComponentName:               "v" + version + "/windowszip/v" + version + "-1int.zip",
-			common.WindowsArtifactAzureStackComponentName:     "v" + version + common.AzureStackSuffix + "/windowszip/v" + version + common.AzureStackSuffix + "-1int.zip",
+			common.WindowsArtifactAzureStackComponentName:     "v" + version + "/windowszip/v" + version + "-1int.zip",
```

Comment:
Just to confirm, starting on `v1.21` we remove the suffix?

Reply:
Yes, that's the version when in-tree cloud provider deprecates and external cloud provider comes into the picture